Tests/UnitTests/TestHelpers/SnapshotTesting+Extensions.swift

```diff
@@ -155,8 +155,8 @@ private let traits: UITraitCollection = .init(displayScale: 1)
 
 #endif
 
-private let perceptualPrecision: Float = 0.97
-private let timeout: DispatchTimeInterval = .seconds(5)
+private let perceptualPrecision: Float = 0.94
```

Comment:
Just out of curiosity, what failed if we keep it at 97?

Reply:
No idea, the diff produced showing the inconsistency was completely black to the naked eye. So just very tiny color differences.